### DISCUSSION TITLE
Benefit Application: I can't upload a file, the rejection reason isn't clear, and on the phone the button is just disabled

### DISCUSSION THREAD

**Discussion by Dr4gon** [community/community#58725] [GitHub Education]
### Hi everyone,

Dear GitHub Community,

I'm thoroughly confused and baffled by this system as an engineer with 12 years of experience:

1. I can only take a photo while applying as a teacher for a school? Why? I had to flip my laptop to the side, print my lecturer contract, and add the signature half below the first page. Of course it's unreadable. What did u expect? 
2. The GPS coordinates are supposed to tell you what? That in times of Corona everybody is working from everywhere. That developers from the dawn of time have worked remotely? This makes no sense whatsoever to me.
3. I get a generic response saying: Common verification problems and suggested solutions are documented here. Come one people. In software development we always provide a proper error message and a code for the techies. 
4. I tried on my iPhone and the button just stays disabled although I have activated your GPS tracking. No further feedback, error message, or anything.

I would've have written that in public to give you the chance to fix my verification. Since you don't provide a mail address or I can't find it, here we are.

I feel like HULK SMASH right now. And to me this is truly sad for an organization that's supposed to be an role model for software engineers all around the world.

How am I able to get the verification, so I can teach properly? 🫣

Best regards,
De Long

**Comment by mai-soup** [comment]
Someone help the man out, I need my sensei
